src/zcbor_decode.c

```diff
@@ -404,24 +404,30 @@ static bool str_start_decode(zcbor_state_t *state,
 	return true;
 }
 
+
+static bool str_overflow_check(zcbor_state_t *state, size_t len)
+{
+	/* Casting to size_t is safe since value_extract() checks that
```

Comment:
I don't get this comment, where do we perform `value_extract` ? Given that we pass a `state` and not a `str` I would argue that this function deserves a docstring.